ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -1995,9 +1995,8 @@ objects:
           gsutil rb gs://${INFRA_ID}-bootstrap-ignition
           gcloud deployment-manager deployments delete -q ${CLUSTER_NAME}-bootstrap
         elif [[ "${CLUSTER_TYPE}" == "openstack" ]]; then
-          # TODO: destroy bootstrap resources
-          echo "openstack bootstrap resources were not destroyed!"
-          echo "openstack bootstrap complete"
+          # Destroy bootstrap resources
+          ansible-playbook -i inventory-upshift.yaml ./down-03_bootstrap.yaml
```

Comment:
Since we are making the call here, do we need to do it again in the teardown container? (line 2299 of this file)

Reply:
Also, I think this is "inventory.yaml" not "inventory-upshift.yaml". At least the file included in with the playbooks is https://github.com/openshift/installer/blob/master/upi/openstack/inventory.yaml
Or is this inventory file a different one?

Reply:
oh yeah you are right, that is a mistake